### PR TITLE
Add Exercise 2

### PR DESCRIPTION
**Pull request by nguyenvanman**

### CODE REVIEW

File: Exercise_2/RootObject.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace Exercise_2
+{
+    public class RootObject
```
Comment (voquanghoa) [inline Exercise_2/RootObject.cs]: Rename class to `User`

File: Exercise_2/Program.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        static void Main(string[] args)
+        {
+            foreach (var listItem in GetListObjects())
+            {
+                Console.WriteLine(listItem.ToString());
+            }
+            Console.ReadKey();
+        }
+
+        static List<RootObject> GetListObjects()
```
Comment (voquanghoa) [inline Exercise_2/Program.cs]: `GetUsers`

File: Exercise_2/IJsonPlaceHolder.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+using System.Text;
+using System.Threading.Tasks;
+using RestSharp;
+using Retrofit.Net.Attributes.Methods;
+
+namespace Exercise_2
+{
+    public interface IJsonPlaceHolder
+    {
+        [Get("/posts")]
```
Comment (voquanghoa) [inline Exercise_2/IJsonPlaceHolder.cs]: Something went wrong here

File: Exercise_2/RootObject.cs (hunk 1 above)
Comment (nguyenvanman) [reply Exercise_2/RootObject.cs]: Done

File: Exercise_2/Program.cs (hunk 2 above)
Comment (nguyenvanman) [reply Exercise_2/Program.cs]: Done

File: Exercise_2/IJsonPlaceHolder.cs (hunk 3 above)
Comment (nguyenvanman) [reply Exercise_2/IJsonPlaceHolder.cs]: Cái này sai chỗ nào vậy anh @@

**Pull request merged**